heartbeat/beater/heartbeat.go

```diff
@@ -182,10 +182,8 @@ func (bt *Heartbeat) RunStaticMonitors(b *beat.Beat) (stop func(), err error) {
 
 // RunCentralMgmtMonitors loads any central management configured configs.
 func (bt *Heartbeat) RunCentralMgmtMonitors(b *beat.Beat) {
-	mons := cfgfile.NewRunnerList(management.DebugK, bt.dynamicFactory, b.Publisher)
-	reload.Register.MustRegisterList(b.Info.Beat+".monitors", mons)
```

Comment:
Does removing this give the same behaviour as before?

Reply:
Not sure what you mean?

A while ago, I think on one of the V2 meetings, someone told me that agent will only even care about the `input` interfaces, and the other ones (like `*.monitors`) won't be configured, so I'm going off of that.

Reply:
I think that makes sense. A regular heartbeat config file uses `monitors` but I think agent only uses `inputs` so I think that's fine.